.circleci/config.yml

```diff
@@ -76,14 +43,68 @@ jobs:
       - run:
           name: Test
           command: npm run test
+
+  deploy-staging:
+    docker:
+      - image: cimg/base:stable
+    resource_class: medium+
+    steps:
+        - vault/get-secrets:
+            template-preset: "aws-push-artifacts"
+        - run:
+            name: Install awscli
+            command: |
+              sudo apt-get update
+              sudo apt-get install python3-pip python3-dev
+              sudo pip3 install awscli
+        - run:
+            name: Build
+            command: |
+                  REACT_APP_BACKEND_BASE_URL=$BACKEND_BASE_URL_TEST \
+                  REACT_APP_SLACK_CLIENT_ID=$SLACK_CLIENT_ID_TEST \
+                  npm run build
+        - run:
+            name: Deploy apps to staging
+            command: |
+              STATIC_S3_BASE="s3://cf-apps-static-dev/apps-test-$CIRCLE_SHA1" \
+                  STATIC_JIRA_S3_BASE="s3://cf-apps-static-dev/apps-test-$CIRCLE_SHA1/jira" \
+                  REACT_APP_BACKEND_BASE_URL=$BACKEND_BASE_URL_TEST \
+                  REACT_APP_SLACK_CLIENT_ID=$SLACK_CLIENT_ID_TEST \
+                  STAGE='test' npm run deploy:test
+        - run:
+            name: Invalidate Slack staging cloudfront distribution
+            command: aws cloudfront create-invalidation --distribution-id $SLACK_TEST_CLOUDFRONT_DIST_ID --paths "/*"
+
+  deploy-prod:
+    docker:
+      - image: cimg/base:stable
+    resource_class: medium+
+    steps:
+      - vault/get-secrets:
+          template-preset: "aws-push-artifacts"
+      - run:
+          name: Install awscli
+          command: |
+            sudo apt-get update
+            sudo apt-get install python3-pip python3-dev
+            sudo pip3 install awscli
+      - run:
+          name: Build
+          command: |
+                REACT_APP_BACKEND_BASE_URL=$PROD_APP_SLACK_BACKEND_BASE_URL \
+                REACT_APP_SLACK_CLIENT_ID=$REACT_APP_SLACK_CLIENT_ID \
```

Comment:
Maybe also replace these client id vars to avoid the same problems?

Reply:
These two are the same but will change them just in case 😅